docs/Using-Fleet/Vulnerability-Processing.md

```diff
@@ -200,4 +201,40 @@ If there's a match, we remove the false positive from the list.
 
 - CentOS 6, 7 and 8.
 
+## Coverage
```

Comment:
Same as my suggestion above. I think we can move the new "Coverage" section up so that it is the second section of the docs (following "What to expect"). What do you think?

Reply:
I like this idea, changes should have been addressed in the latest revision.